devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-native.ftl

```diff
@@ -17,6 +17,6 @@
 FROM registry.access.redhat.com/ubi8/ubi-minimal
 WORKDIR /work/
 COPY ${build_dir}/*-runner /work/application
-RUN chmod 775 /work
```

Comment:
My guess is that you should keep this line too and have both.

Reply:
@gsmet What about chmod -R 775 /work for recursiv setting?

Reply:
But would be to follow your advice.

Reply:
Yeah, I'm not a big fan of making things executable recursively. Users could add files there by mistake.